skglm/experimental/__init__.py

```diff
@@ -1,5 +1,9 @@
+from .reweighted import IterativeReweightedL1, _L23_objective, _L23_derivative
```

Comment:
object prefixed with _ are "private" and should not be exposed. Can you find a workaround? Which piece of code needs them like this?